src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Compiler/ValidateWorkflowsPass.php

```diff
@@ -16,6 +16,7 @@
 use Symfony\Component\DependencyInjection\Exception\RuntimeException;
 use Symfony\Component\Workflow\Validator\DefinitionValidatorInterface;
 use Symfony\Component\Workflow\Validator\StateMachineValidator;
+use Symfony\Component\Workflow\Validator\UniqueTransitionNameValidator;
```

Comment:
There is no reference to symfony/workflow at all in the composer.json of the framework bundle. Should I still add one?